app/modules/intelligence/prompts_provider/agent_prompts_provider.py

```
@@ -0,0 +1,61 @@
+            description = AnthropicAgentPrompts.get_anthropic_task_prompt(task_id)
+        elif preferred_llm == AgentRuntimeLLMType.OPENAI.value.lower():
+            description = OpenAIAgentPrompts.get_openai_task_prompt(task_id)
+        else:
+            return None
+
+        if description and kwargs:
+            try:
+                # Format the description string using the processed kwargs\
+                description = description.format(**kwargs)
+            except KeyError as e:
+                raise ValueError(f"Missing key in kwargs for formatting: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)